app/scripts/modules/core/src/search/infrastructure/RecentlyViewedItems.tsx

```diff
@@ -70,12 +85,22 @@ export class RecentlyViewedItems extends React.Component<{}, IRecentlyViewedItem
   }
 
   public render() {
+    const { Component } = this.props;
+
     return (
-      <SearchResultPods
-        results={this.state.recentItems}
-        onRemoveItem={this.handleRemoveItem}
-        onRemoveProject={this.handleRemoveProject}
-      />
+      Component ? (
+        <Component
+          results={this.state.recentItems}
+          onRemoveItem={this.handleRemoveItem}
+          onRemoveProject={this.handleRemoveProject}
+        />
+       ) : (
+        <SearchResultPods
```

Comment:
I *really* didn't want to hard code a default component here, since it's so easy to just pass a ref to `SearchResultPods`, but unfortunately there is still *one* angular template where we use `<recently-viewed-items>` and from what I can tell it's impossible to pass a component ref via a template. If someone has a good way of doing it please let me know and I'll rip this out.